### PR TITLE
Add mysql connector

### PR DESCRIPTION
**Pull request by dnadein**
Init pull request for mysql connector and mysql sqlalchemy packages

### CODE REVIEW

File: lib/dl_formula/pyproject.toml [hunk 1]
```diff
@@ -23,7 +23,7 @@ dynamic-enum = {path = "../dynamic_enum"}
 [tool.poetry.group]
 [tool.poetry.group.tests.dependencies]
 pytest = ">=7.2.2"
-mysqlclient = ">=1.4.6"
+pymysql = "==1.1.0"
 oracledb = ">=1.3.1"
```
Comment (asnytin) [inline lib/dl_formula/pyproject.toml:27]: we shouldn't need these deps in a base dl_formula package
nothing related to specific connectors (pg, ch, oracle...)

File: lib/dl_connector_mysql/Makefile
Comment (asnytin) [inline lib/dl_connector_mysql/Makefile:1]: ?

Comment (dnadein) [reply lib/dl_connector_mysql/Makefile:1]: Removed this file

File: lib/dl_formula/pyproject.toml (hunk 1 above)
Comment (dnadein) [reply lib/dl_formula/pyproject.toml:27]: Seems that you're right. Will remove this dep and update lock file.

**Review by asnytin — APPROVED** [review]

**Pull request merged**